hhs_oauth_server/settings/test.py

```diff
@@ -26,4 +26,10 @@
     'OAUTH2_BACKEND_CLASS': 'apps.dot_ext.oauth2_backends.OAuthLibSMARTonFHIR',
     'ALLOWED_REDIRECT_URI_SCHEMES': ['https', 'http']
 }
+
+REST_FRAMEWORK = {
+    'DEFAULT_THROTTLE_RATES': {
+        'token': '1/day',
```

Comment:
I'm confused as to how this is not affecting our other integration tests. Aren't they exceeding this limit?

Also, I think I'd be more comfortable setting this to 1/minute. I know that this should be reset on application restart, but this is one of those things you could accidentally trigger and make things unnecessarily difficult for yourself.

Reply:
It may be affecting integration tests, are they run during the build process? I'll attempt to remove this from the test settings and set it only for the required tests.